crates/sui-framework/packages/sui-system/tests/staking_pool.move

```diff
@@ -26,7 +29,7 @@ module sui_system::staking_pool_tests {
     }
 
     #[test]
-    #[expected_failure(abort_code = 1, location = sui_system::staking_pool)]
+    #[expected_failure(abort_code = 1, location = ::sui_system::staking_pool)]
```

Comment:
Can you use a constant here instead? And in other tests too.
```suggestion
    #[expected_failure(abort_code = ::sui_system::staking_pool::EWrongPool)]
```

Reply:
done